torch/onnx/symbolic_opset9.py

```diff
@@ -1549,19 +1549,28 @@ def clamp(g, self, min, max):
     elif sym_help._is_none(max):
         return clamp_min(g, self, min)
     else:
-        min = _parse_arg(min, 'f')
-        max = _parse_arg(max, 'f')
-        return g.op("Clip", self, min_f=min, max_f=max)
+        min = _parse_arg(min, 'v')
+        max = _parse_arg(max, 'v')
+        if sym_help._get_tensor_rank(min) == 0 and sym_help._get_tensor_rank(max) == 0:
+            return g.op("Clip", self, min_f=_parse_arg(min, 'f'), max_f=_parse_arg(max, 'f'))
```

Comment:
`_parse_arg(min, 'f')` only works if `min` is a Constant (value known at export time). onnx opset 9 cannot support `min` as tensor. It will need at least opset 11. Could you add the message similar to https://github.com/pytorch/pytorch/blob/8d7faa2af88179d1be37b1ceccc301c62b61ebd4/torch/onnx/symbolic_opset9.py#L1030 
 (this is a similar case where opset 9 `onnx::Pad` only works with `pad` value being constant)

Reply:
`Clip(tensor, min, max)` only supports scalar min and max, so I have to use `Min(Max(tensor, min), max)` instead. `Min` and `Max` support dynamic tensors and from opset 8 also allow broadcasting. So we can support both forms of `clamp`, albeit not using the `Clip` operator.

As you point out though, `_get_tensor_rank == 0` isn't for opset 9 here, so replaced with `_is_constant`. Now `test_clip_dyn` works from opset 8 and up.

Reply:
@BowenBao PTAL